src/vfio_device.rs

```diff
@@ -968,3 +968,69 @@ impl Drop for VfioDevice {
         self.container.put_group(self.group.clone());
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use kvm_ioctls::Kvm;
+    use kvm_bindings::kvm_device_type_KVM_DEV_TYPE_VFIO;
+
+    // Real VFIO device with VFIO_DEVICE_PATH is used to test
+    const VFIO_DEVICE_PATH: &str = "/sys/bus/pci/devices/0000:00:14.0/";
```

Comment:
I think the `test_vfio_device` there is useful, but it needs to be documented  a little more:

1. Specify that this PCI path is used by a test that's ignored by default
1. Describe how one could pick a local device for testing
1. Describe how this device should unbound from its regular driver and bound back to vfio
1. Descrive how the test can then be enabled back

Reply:
I have updated with more document :)